app/models/accounting/transaction.rb

```diff
@@ -44,12 +44,11 @@ class Accounting::Transaction < ApplicationRecord
   validates :amount, presence: true, unless: :interest?, if: :managed?
   validates :accounting_account_id, presence: true, unless: :interest?, if: :managed?
   validate :respect_closed_books_date, if: :user_created
-  with_options if: ->(txn) { txn.qb_object_subtype == "Check" && txn.user_created } do
-    validates :check_number, presence: true
-  end
-  # validate that all disbursements created in Madeline's transaction form have a vendor
   with_options if: ->(txn) { txn.loan_transaction_type_value == "disbursement" && txn.user_created } do
-    validates :qb_vendor_id, presence: true
+    validate :disbursement_fields
```

Comment:
These `with_options` blocks are I believe intended to be used if you have bunch of validations that should only apply if those conditions are met. It seems odd to use it when there is just one validation in each. 

Could we just do:

```
validate :disbursement_fields, if: ->(txn) { txn.loan_transaction_type_value == "disbursement" && txn.user_created }
```

You could also shorten things by adding an boolean `disbursement?` method, which I think is a good idea since it's best to check against the literal string `"disbursement"` in only one place to avoid errors.

So that would make it

```
validate :disbursement_fields, if: ->(txn) { txn.disbursement? && txn.user_created }
```

Which is nice and neat!